### PR TITLE
Consolidate dashboard diagnostics documentation

### PR DESCRIPTION
**Pull request by scharlottej13**
These edits are tangentially related to improving SEO. Currently, https://docs.dask.org/en/stable/diagnostics-distributed.html ranks in the top spot for when people search for the diagnostics dashboard. Instead, I think we'd rather people find https://docs.dask.org/en/stable/dashboard.html. This PR consolidates the dashboard information onto https://docs.dask.org/en/stable/dashboard.html and updates some of the internal documentation references accordingly.

@ncclementi, would this be OK with you? Asking since it sounded like you had some thoughts on preserving the dashboard content in diagnostics-distributed [#8648 here](https://github.com/dask/dask/pull/8648#issuecomment-1028044453).

cc @jacobtomlinson @jsignell @jrbourbeau

### CODE REVIEW

File: docs/source/dashboard.rst [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
 
-The address of the dashboard will be displayed if you are in a Jupyter Notebook,
-or, if you are in a terminal or IPython, it can be queried from ``client.dashboard_link``. By default, when starting a scheduler 
-on your local machine the dashboard will be served at ``http://localhost:8787/status``. You
-can type this address into your browser to access the dashboard, but may be directed 
-elsewhere if port 8787 is taken. You can also configure the address by passing options to the 
-scheduler, see ``dashboard_address`` in `LocalCluster <https://docs.dask.org/en/stable/deploying-python.html#reference>`__
+By default, when starting a scheduler on your local machine the dashboard will be served at ``http://localhost:8787/status``. You can type this address into your browser to access the dashboard, but may be directed 
+elsewhere if port 8787 is taken. You can also configure the address using the ``dashboard_address``
+parameter (see :meth:`LocalCluster <distributed.deploy.local.LocalCluster>`).
```
Comment (jacobtomlinson) [inline docs/source/dashboard.rst:39]: Should this be `class` instead?

```suggestion
parameter (see :class:`LocalCluster <distributed.deploy.local.LocalCluster>`).
```

File: docs/source/dashboard.rst [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
+            style="margin: 0 auto 20px auto; display: block;"
+            allowfullscreen>
+    </iframe>
+
+Profiling parallel code can be challenging, but the interactive dashboard provided with
+Dask's :doc:`distributed scheduler <scheduling>` makes this easier with live monitoring
+of your Dask computations. The dashboard is built with `Bokeh <https://docs.bokeh.org>`_
+and will start up automatically, returning a link to the dashboard whenever the scheduler is created.
+
+Locally, this is when you create a :meth:`Client <distributed.client.Client>` and connect the scheduler:
```
Comment (jacobtomlinson) [inline docs/source/dashboard.rst:23]: Should also be `class`?

**Review by jacobtomlinson — APPROVED** [review]
This looks great, couple of small comments.

File: docs/source/dashboard.rst (hunk 1 above)
Comment (scharlottej13) [reply docs/source/dashboard.rst]: yes! changed

File: docs/source/dashboard.rst (hunk 2 above)
Comment (scharlottej13) [reply docs/source/dashboard.rst]: yup! changed

**Comment by scharlottej13** [comment]
> This looks great, couple of small comments.

Thanks @jacobtomlinson! Just made those changes

**Review by jrbourbeau — APPROVED** [review]
Thanks @scharlottej13 for the updates and @jacobtomlinson for reviewing

**Pull request merged**